lib/improver/tests/utilities/test_RegridLandSea.py

```diff
@@ -0,0 +1,333 @@
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Unit tests for the RegridLandSea class from spatial.py."""
+
+import unittest
+import numpy as np
+
+import iris
+from iris.tests import IrisTest
+from iris.coords import DimCoord
+from iris.cube import Cube
+from iris.util import squeeze
+
+from improver.tests.nbhood.nbhood.test_BaseNeighbourhoodProcessing import (
+    set_up_cube)
+from improver.utilities.spatial import (
+    RegridLandSea, OccurrenceWithinVicinity)
+from improver.utilities.warnings_handler import ManageWarnings
+from improver.grids import ELLIPSOID
+
+
+class Test__init__(IrisTest):
+    """Tests for the initiation of the RegridLandSea class."""
+
+    def test_basic(self):
+        """Test that instantiating the class results in an object with
+        expected variables."""
+        expected_members = {'nearest_cube': None,
+                            'input_land': None,
+                            'output_land': None,
+                            'output_cube': None}
+        result = RegridLandSea()
+        members = {attr: getattr(result, attr) for attr in dir(result)
+                   if not callable(getattr(result, attr))
+                   and not attr.startswith("__")}
+        regridder = members.pop('regridder')
+        vicinity = members.pop('vicinity')
+        self.assertDictEqual(members, expected_members)
+        self.assertTrue(isinstance(regridder, iris.analysis.Nearest))
+        self.assertTrue(isinstance(vicinity, OccurrenceWithinVicinity))
+
+    def test_extrap_arg(self):
+        """Test with extrapolation_mode argument."""
+        result = RegridLandSea(extrapolation_mode="mask")
+        regridder = getattr(result, 'regridder')
+        self.assertTrue(isinstance(regridder, iris.analysis.Nearest))
+
+    def test_extrap_arg_error(self):
+        """Test with invalid extrapolation_mode argument."""
+        msg = "Extrapolation mode 'not_valid' not supported"
+        with self.assertRaisesRegex(ValueError, msg):
+            RegridLandSea(extrapolation_mode="not_valid")
+
+    def test_vicinity_arg(self):
+        """Test with vicinity_radius argument."""
+        result = RegridLandSea(vicinity_radius=30000.)
+        vicinity = getattr(result, 'vicinity')
+        self.assertTrue(isinstance(vicinity, OccurrenceWithinVicinity))
```

Comment:
This doesn't appear to test anything different to the default 25000 test, i.e. I don't know what radius is going to be used from this test. Could you assert the radius set within OccurenceWithinVicinity to ensure it is creating a suitable instance of it (e.g, the value is being passed on) or remove the test if that is not possible.